src/adapter/src/coord/timestamp_oracle/postgres_oracle.rs

```diff
@@ -109,20 +84,24 @@ impl PostgresTimestampOracleConfig {
 
     /// Returns a new instance of [`PostgresTimestampOracleConfig`] with default tuning.
     pub fn new(url: &str, metrics: Arc<Metrics>) -> Self {
-        PostgresTimestampOracleConfig {
-            url: url.to_string(),
-            metrics,
+        let dynamic = DynamicConfig {
             // TODO(aljoscha): These defaults are taken as is from the persist
```

Comment:
Probably fine to drop this TODO since we're making them configurable

Reply:
I haven't thought about what these values should be, so seems good to still leave this TODO